laws/shared/src/main/scala/algebra/laws/IsSerializable.scala

```diff
@@ -0,0 +1,12 @@
+package algebra.laws
+
+import scala.util.DynamicVariable
+
+/**
+  * Object with a dynamic variable that allows users to skip the
+  * serialization tests for certain instances.
+  */
+object IsSerializable {
```

Comment:
should we make this `private[laws]` and just let people interact with this via the `withoutSerialization`?